MonitoringSystem/Client/MonitoringReporter.py

```diff
@@ -63,7 +63,7 @@ def processRecords( self ):
     It consumes all messaged from the MQ (these are failover messages). In case of failure, the messages
     will be inserted to the MQ again.
     """
-    result = createConsumer( self.__monitoringType )
+    result = createConsumer( "Monitoring::Queue::%s" % self.__monitoringType )
```

Comment:
No, It was always like this, but I hot fixed in the machine when I tried out. In principle we are not using MQ in production. We will have a downtime and I would like to setup the machinery to make sure we will not loose data.